sdk/storage/Azure.Storage.DataMovement.Blobs/src/BlobsStorageResourceProvider.cs

```diff
@@ -300,17 +245,22 @@ internal async Task<StorageResource> FromDestinationInternalHookAsync(
         /// <param name="options">
         /// Options for creating the storage resource.
         /// </param>
+        /// <param name="cancellationToken"></param>
```

Comment:
NIT - update to
```
<param name="cancellationToken">
        /// Optional <see cref="CancellationToken"/> to propagate
        /// notifications that the operation should be cancelled.
        /// </param>
        ```
        
        same for the rest

Reply:
Done